packages/cli/src/hooks/init/load-plugins.ts

```diff
@@ -1,12 +1,19 @@
 import { Hook as OclifHook } from '@oclif/core'
 import { initHook } from '@preevy/cli-common'
+import { telemetryEmitter } from '@preevy/core'
 
 const wrappedHook: OclifHook<'init'> = async function wrappedHook(...args) {
   try {
+    const { id } = args[0]
```

Comment:
IMO this function shouldn't include any logic besides wrapping with try..catch, you can move the id check to the hook itself.